### ISSUE TITLE
Post webmentions to non-Tumblr Disqus threads

### ISSUE THREAD

**Issue by fluffy-critter** [snarfed/bridgy#855]
From the commentary on #639 I think I’m looking for the opposite of what @billbennettnz is looking for - I use Disqus for my comment threads on my blog, and I’d love incoming webmentions to show up as comments on said threads, at least on the pages where Disqus threads exist. My understanding is that this is the level of support brid.gy has for Tumblr in the first place - but I’m not finding any documentation on how the Tumblr integration actually works or how to set it up outside of Tumblr itself.

Is this something that’s possible or am I completely misunderstanding things?

(If this isn’t easy to do, my fallback plan is to simply use webmention.io’s API to embed mentions via some basic Javascript, which is *adequate* but has some UX compromises.)

**Comment by snarfed** [comment]
hi! it sounds like you're looking for a general purpose library or service to accept and display all incoming webmentions? ie alongside your existing disqus integration? yeah, that's definitely outside bridgy's scope. its goal is to bridge webmentions with silos, whether social silos or hosted blog silos. (tumblr is a bit of both.)

if you're asking for bridgy to accept incoming webmentions and propagate them _into_ disqus, that is indeed an interesting feature request, and yes, more like the opposite of #639. i probably wouldn't prioritize on implementing that myself right now, but PRs are welcome!

**Comment by fluffy-critter** [comment]
> if you're asking for bridgy to accept incoming webmentions and propagate them into disqus, that is indeed an interesting feature request

Yeah, that's what I was asking for. Sorry if I was unclear! The explanation of what brid.gy is and does is a bit vague and I'm never sure what role it actually plays in a webmention ecosystem. :)  But if its role is to convert silo'd mentions into webmentions for consumption by other things that support webmention already, then it sounds like a better approach is to write a webmention adapter for Disqus instead, which could be called by webmention.io's webhook thing (for example).

Anyway, if that's the case then it sounds like this functionality doesn't actually belong in brid.gy, so feel free to close this issue.

**Comment by snarfed** [comment]
got it! bridgy actually converts silo interactions both from _and_ to native webmentions, both directions. most people know it for the former, eg converting twitter @-replies and likes to webmentions, but it also does the reverse for hosted blogs, eg converting webmentions to native comments on blogger and wordpress blogs. background: https://brid.gy/about#blogs

it looks like disqus's API is pretty comprehensive, and allows creating comments and replies to comments, https://disqus.com/api/docs/#resources , so i'm happy to keep this open as a feature request.

**Comment by snarfed** [comment]
related: #270
